lib/modules/plugin_cmd/index.js

```diff
@@ -0,0 +1,64 @@
+let fs = require('./../../core/fs.js');
+let utils = require('./../../utils/utils.js');
+let async = require('async');
+class PluginCommand {
+  constructor(embark, config) {
+    this.embark = embark;
+    this.config = config;
+    this.embarkConfig = fs.readJSONSync(this.config.embarkConfigFile);
+    this.registerCommand();
+  }
+  registerCommand() {
+    const self = this;
+    self.embark.registerConsoleCommand((cmd, _options) => {
+      let cmdArray = cmd.split(' ');
+      cmdArray = cmdArray.filter(cmd => cmd.trim().length > 0);
+      let cmdName = cmdArray[0];
+      return {
+        match: () =>  cmdName === 'plugin',
+        process: (callback) => {
+          if(cmdArray.length < 3 || cmdArray[1] !== 'install' || typeof cmdArray[2] === 'undefined') {
```

Comment:
Should we not reuse the library manager?
@iurimatias

Reply:
If we want to use the library manager, we need to parse the version and package from the ```npm install```. It can get complicated to parse the command if they are installing packages from github for example